go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -495,84 +511,198 @@ func newSharedGatewayOpenFlowManager(patchPort, macAddress, gwBridge, gwIntf str
 				"actions=ct(commit,zone=%d,nat,table=2)",
 				defaultOpenFlowCookie, HostMasqCTZone))
 	}
+	return dftFlows, nil
+}
 
-	// add health check function to check default OpenFlow flows are on the shared gateway bridge
-	ofm := &openflowManager{
-		gwBridge:    gwBridge,
-		physIntf:    gwIntf,
-		patchIntf:   patchPort,
-		ofportPhys:  ofportPhys,
-		ofportPatch: ofportPatch,
-		flowCache:   make(map[string][]string),
-		flowMutex:   sync.Mutex{},
-		flowChan:    make(chan struct{}, 1),
+func flowsForExGWBridge(ofPortPhys, bridgeMacAddress, ofPortPatch string) []string {
```

Comment:
since these flows are really a subset of the normal flows on default bridge, you can just consolidate it to a common function and call it in both places. Then we only need to change the code for the flows in one place.